deploy/operator/setup_assisted_operator.sh

```diff
@@ -172,10 +184,31 @@ EOCR
 }
 
 function from_index_image() {
+  default_ocp_versions="${__dir}/../../data/default_ocp_versions.json"
```

Comment:
I think this should be received from the upper-level function (`deploy.sh`) and `setup_assisted_operator.sh` can support a received `OPENSHIFT_VERSIONS` (unmodified) as it already does for u/s deployment.

Reply:
done